website/index.html

```diff
@@ -483,7 +491,7 @@ <h2>Like ClickHouse?</h2>
    target="_blank"
 ><div id="github">Fork me on GitHub</div></a>
 
-<script type="text/javascript" src="//yastatic.net/jquery/3.1.1/jquery.min.js"></script>
+<script type="text/javascript" src="https://yastatic.net/jquery/3.1.1/jquery.min.js"></script>
```

Comment:
if url scheme is omitted it's taken from current page url, so this change doesn't do much except adding 6 bytes to content

Reply:
It's required to opening from `file://` to work.